database/migrations/2025_05_12_183803_add_req_serial_bool_to_models_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('models', function (Blueprint $table) {
+            $table->boolean( 'require_serial')->after('category_id')->default(0)->nullable();
```

Comment:
We _shouldn't_ need `nullable` as well as `default(0)`? We can mess around with that and see what's going on there. @snipe says we might actually need it for other DB engines...